### DISCUSSION TITLE
Zeek Correlations: Outbound Connections to IP address without DNS lookup

### DISCUSSION THREAD

**Discussion by presianbg** [Security-Onion-Solutions/securityonion#7605] [Ideas] [answered]
Hi,

Few days ago I've listed to the [SANS Daily Network Security Podcast](https://isc.sans.edu/podcastdetail.html?id=7918) in which  Johannes Ullrich talked about [Raccoon Stealer: “Trash panda” abuses Telegram](https://decoded.avast.io/vladimirmartyanov/raccoon-stealer-trash-panda-abuses-telegram/) .

Johannes continued his summary with a very good advice - " The URLs here appeared to be just IP addresses, not hostnames. A zeek script detecting outbound connections to IP addresses that did not get returned as DNS responses... something good to have." 

I followed this idea and looked for such scripts available, but I only found this article by David Hoelzer:

[Zeek Correlations: Outbound Connections](https://showmethepackets.com/index.php/2019/09/19/zeek-correlations-outbound-connections/):
![image](https://user-images.githubusercontent.com/2163486/158966318-df6eb9e5-7b04-4293-8544-29ac8f518583.png)

Have anyone in the community have something like this in-place utilizing zeek ?
Any thoughts on this topic is welcomed also, how to implement such detection in best way possible ?

Cheers,
PY

**Comment by dougburks** [comment, marked as answer]
Besides the Zeek script in the article, another option might be to leverage the `http.dottedquadhost` rules that are already in the Emerging Threats NIDS ruleset.
